public/src/style.scss

```diff
@@ -652,4 +664,4 @@ nav.flexbox {
     width: 20px;
     text-align: center;
     color: slategray;
-}
+}
```

Comment:
Please add a new line. GitHub doesn't like that files end without a newline and saving a byte is not that big of a deal.